sdcm/cluster_k8s/__init__.py

```diff
@@ -95,6 +95,13 @@ def setup_minikube_ubuntu18(node: cluster.BaseNode, kubectl_version: str, miniku
             echo fs.aio-max-nr=1048576 >> /etc/sysctl.d/99-sct-minikube.conf
             echo net.ipv4.ip_forward=1 >> /etc/sysctl.d/99-sct-minikube.conf
             echo net.ipv4.conf.all.forwarding=1 >> /etc/sysctl.d/99-sct-minikube.conf
+            echo kernel.sched_tunable_scaling=0 >> /etc/sysctl.d/99-sct-minikube.conf
+            echo kernel.sched_min_granularity_ns=500000 >> /etc/sysctl.d/99-sct-minikube.conf
+            echo kernel.sched_wakeup_granularity_ns=450000 >> /etc/sysctl.d/99-sct-minikube.conf
+            echo kernel.sched_latency_ns=1000000 >> /etc/sysctl.d/99-sct-minikube.conf
+            echo kernel.sched_autogroup_enabled=0 >> /etc/sysctl.d/99-sct-minikube.conf
+            echo kernel.numa_balancing=0 >> /etc/sysctl.d/99-sct-minikube.conf
+            echo vm.swappiness=1 >> /etc/sysctl.d/99-sct-minikube.conf
```

Comment:
Can you explain how these tunings relate to upgrade test?

Reply:
These are leftovers from  finding solution to make truncate work